src/applications/facility-locator/sass/facility-locator.scss

```diff
@@ -292,7 +292,6 @@ $facility-locator-shadow: rgba(0, 0, 0, 0.5);
 
       .use-my-location-link {
         text-decoration: none;
-        margin: 0 0 0 20px;
```

Comment:
This change will eliminate the left margin here. Why is this needed?

![Find_VA_Locations___Veterans_Affairs](https://user-images.githubusercontent.com/80267/110895662-2b82e200-82c8-11eb-9805-a45c4264f840.png)

Reply:
well  this margin is making the clear input be off
<img width="1346" alt="Screen Shot 2021-03-11 at 11 28 22 PM" src="https://user-images.githubusercontent.com/100468/110896706-bf9d7b00-82c1-11eb-877c-21c1b51658b1.png">

do we actually need the margin?